src/structs.jl

```diff
@@ -224,54 +224,38 @@ tax(T::Tax) = T.tax
 
 
 struct ScalarNest
-    name::Symbol
-    subindex::Any
+    name::String
+    base_name::String
     elasticity::MPSGEquantity
-    function ScalarNest(name::Symbol, elasticity::MPSGEquantity; subindex = missing)
-        new(name, subindex, elasticity)
+    function ScalarNest(name::String, base_name::String, elasticity::MPSGEquantity)
+        new(name, base_name, elasticity)
     end
 end
 
 
-base_name(N::ScalarNest) = N.name
-name(N::ScalarNest) = ismissing(subindex(N)) ? N.name : Symbol(N.name,"_",join(subindex(N),"_"))
+base_name(N::ScalarNest) = Symbol(N.base_name)
+name(N::ScalarNest) = Symbol(N.name)
 elasticity(N::ScalarNest) = N.elasticity
-subindex(N::ScalarNest) = N.subindex
 
 struct IndexedNest{N} <: AbstractArray{ScalarNest, N}
-    name::Symbol
+    name::String
     subsectors::DenseAxisArray{ScalarNest,N}
     index::Any
-    function IndexedNest(name::Symbol, elasticity::MPSGEquantity, index)
```

Comment:
replaced by nest.jl in macros
It was not able to handle indexed nests as well as new version